Animations and Some backsprites for alternate forms

flempa316 opened the pull request:
Animations for:
Alolan Geodude
Alolan Graveler
Alolan Golem
Alolan Marowak
Alolan Vulpix
Alolan Ninetales
Galarian Slowbro
Alolan Persian
Alolan Meowth
Galarian Weezing

Back Sprites For:
Slowbro
Vulpix

Rangi42 commented:
Thanks a lot! Please make the PNGs 2-bit grayscale like the rest, and use normal.pal files for the palettes.

flempa316 commented:
Cool, can do!

flempa316 commented:
@Rangi42 that should be everything!

Rangi42 commented:
Happy holidays, and thank you! Just one more thing: the `frames.asm` are generated by `make` so should not be committed.

flempa316 commented:
Happy holidays to you too! So the animations where I included the frames.asm files are the animations that the `make` process didn't generate the corrects frames file for... if that makes sense. We briefly chatted about it on discord a few months back. Let me know if they should still be removed anyway.

Rangi42 commented:
Oh. Just leave them, I'll see what the issue is later.

flempa316 commented:
Fixed a bunch of mess ups on the monochromes, and some reversed palettes. And removed those frames files - everything should be all set, but let me know if there is anything else
